pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "numalogic-prometheus"
-version = "0.2.4"
+version = "0.2.5a0"
```

Comment:
Let's not change the version here maybe? Will rebase these changes, and have everything in 0.2.5.
What do you think?

Reply:
Okay, works. This would need some changes in manifests though, I will update them just make sure you have those changes before deploying.